Source/Extensions/Blazorise.DataGrid/DataGrid.razor

```diff
@@ -124,10 +124,10 @@
             }
             @if (IsDisplayDataVisible)
             {
-                @if ( VirtualizeManualReadMode )
+                @if (VirtualizeManualReadMode)
                 {
 
-                    <Virtualize @ref="virtualizeRef" TItem="TItem" Context="item" ItemsProvider="HandleVirtualizeReadData">
+                    <Virtualize @ref="virtualizeRef" TItem="TItem" Context="item" ItemsProvider="HandleVirtualizeReadData" ItemSize="(float)63.75">

```

Comment:
Why exactly 63.75? Is this a magical number 😅? Might be a good idea to make it a parameter.

Reply:
Test code. I think I removed it. Most Virtualize parameters will go into a VirtualizeOptions class.

Reply:
Reason why I commit so often, is generally when I have to stop work, and I'm paranoid about loosing work locally and it not being saved remotely...

Reply:
I can relate. There were few times I got burned by failed disk 🔥

But still didn't learn my lessons and committing more😅

Reply:
Haha so you've felt it on your skin...! I think I have lost it only once, and it was quite a bit of work... However it was my fault, not the disk... 😆